.github/workflows/land-blocking.yml

```diff
@@ -123,12 +124,18 @@ jobs:
             ${result.text}
             \`\`\`
             `;
-              let tps = result.metrics.find(m => m.experiment == "all up" && m.metric == "avg_tps").value;
-              let min_tps = 900;
-              if (tps < min_tps) {
-                body += "\n :exclamation: Performance regression is detected on this PR";
-                body += "\n TPS with PR: " + tps + ", this is lower then minimum allowed " + min_tps + " TPS.";
+              // Check CTI exit code for errors
+              if (parseInt(env_vars.CTI_EXIT_CODE) != 0) {
+                body += "\n :exclamation: Cluster Test failed - non-zero exit code for `cti` \n"
```

Comment:
nit - for future iteration, it will be useful to also show people the last 5 lines of cti's stdout/stderr in the github comment (to be expanded by tab though, not inline).